src/components/Controls.vue

```diff
@@ -265,6 +275,11 @@ export default {
 				this.$router.push({ name: 'board.details' })
 			}
 		},
+		clearfilter() {
+			const filterReset = { tags: [], users: [], due: '' }
+			this.$store.dispatch('setFilter', { ...filterReset })
```

Comment:
```suggestion
			dispatch('setFilter', filterReset)
```

Reply:
why?
dispatch is not defined

Reply:
It's not working without ...

Reply:
ah sorry
i was just refering to the resetFilter part ;)